compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -2463,7 +2463,7 @@ OMR::Z::CodeGenerator::doBinaryEncoding()
    static char *disableAlignJITEP = feGetEnv("TR_DisableAlignJITEP");
 
    // Adjust the binary buffer cursor with appropriate padding.
-   if (!disableAlignJITEP && !self()->comp()->compileRelocatableCode() && self()->allowSplitWarmAndColdBlocks())
+   if (!disableAlignJITEP && !self()->comp()->compileRelocatableCode())
```

Comment:
If the ability to split warm and cold blocks has been removed, shouldn't all this code be folded away because `allowSplitWarmAndColdBlocks()` can only return `false` now?

Reply:
I don't think so. I don't know why the restriction was there in the first place. Aligning the JIT-to-JIT entry point has no relation to `allowSplitWarmAndColdBlocks` in my mind.

More to your point this code will be removed as part of #4314 and replaced with a cross-codegen function call.

Reply:
OK.  I was going purely by what the code looked like before (and that now Z in OMR will have different semantics than before).  You're probably right though that the code that this is guarding doesn't really need it.

Reply:
> because allowSplitWarmAndColdBlocks() can only return false now?

`allowSplitWarmAndColdBlocks` no longer exists, so one can argue it can only return true as well. It used to always return true on Z, hence now that it's removed we shouldn't even use the query.